testdata/templates/ssh-git/ssh-git-dc.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: git-server
+spec:
+  replicas: 1
+  selector:
+    name: git-server
+  strategy:
+    type: Rolling
+  template:
+    metadata:
+      labels:
+        name: git-server
+      name: git-server
+    spec:
+      containers:
+      - image: aosqe/ssh-git-server-openshift
+        imagePullPolicy: Always
```

Comment:
Perhaps we should pull image only when it does not exist ?

Reply:
@liangxia Updated, please help review again, thanks